src/utils/filterOrganisationData.ts

```diff
@@ -0,0 +1,15 @@
+interface IOrganisationData {
```

Comment:
This can be moved to interfaces folder -> organisation.ts 
And then used everywhere.